Resources/Prototypes/Entities/Structures/Doors/Windoors/assembly.yml

```diff
@@ -73,3 +73,29 @@
     graph: Windoor
     node: assemblySecure
 
+- type: entity
+  id: WindoorAssemblyClockwork
```

Comment:
https://github.com/space-wizards/space-station-14/assets/96445749/8a2fb189-b717-4237-9348-97fe2b022e9d

the airlock and windoor have different lights when an unauthorized person tries to enter them. Airlock have white and pink flashing, windoor have long yellow flashing. I think the doors should get a yellow light too.

Reply:
that was from the door's deny_unlit.png not working for some reason, and the windoor was missing its deny_unlit.png entirely, i fixed both 👍